Sources/StreamChatUI/ChatMessageList/ChatMessageListCollectionView.swift

```diff
@@ -182,6 +182,13 @@ open class ChatMessageListCollectionView<ExtraData: ExtraDataTypes>: UICollectio
         with changes: [ListChange<_ChatMessage<ExtraData>>],
         completion: ((Bool) -> Void)? = nil
     ) {
+        var shouldScrollToBottom = false
+
+        let bottomEdge = contentOffset.y + frame.size.height
+        if bottomEdge >= contentSize.height {
+            shouldScrollToBottom = true
+        }
+
```

Comment:
Please add a comment here explaining what this magic code does :)

Reply:
Maybe we can move the `isLastCellFullyVisible` from private extension in https://stream-io.atlassian.net/browse/CIS-882 directly to this collection view and reuse it. 
The `isLastCellFullyVisible` function handles `contentInset` well.

Reply:
@skorepak great idea 💯

Reply:
Yes makes sense 👌 Probably with @skorepak extension, the code will be more clear 👍

Reply:
Done ✅